graylog2-web-interface/src/components/permissions/EntityCreateShareFormGroup.tsx

```diff
@@ -101,7 +104,7 @@ const EntityCreateShareFormGroup = ({
       selected_grantee_capabilities: newSelectedCapabilities,
     };
 
-    return EntityShareDomain.prepare(entityType, entityTitle, entityGRN, payload).then((response) => {
+    return EntityShareDomain.prepare(entityType, entityTitle, entityGRN, { ...payload, prepare_request: dependenciesGRN }).then((response) => {
```

Comment:
I think we should just add prepare_request when we defined `const payload...` before the return